### DISCUSSION TITLE
Random Volume Increase

### DISCUSSION THREAD

**Discussion by EthanCannoy** [ppy/osu#28252] [Q&A]
### Type

Game behaviour

### Bug description

Audio randomly gets really loud - seems it may be replaying audio clips.
Happens even during break sections when I am not touching the keyboard/mouse/tablet.
Seems to only happen when playing with low master volume.
On linux using pipewire if that matters.

### Reproduction Info
Basic repro steps: play a beatmap in standard - audio will randomly increase in volume.
Additional info:
* Issue occured multiple times in both standard and taiko
* Issue usually occured between 30s and 60s into a map
* Issue was unrelated to a specific gameplay element
* Both hitsounds and music seemed to increase in volume
* Custom hitsounds were applied when issue occured, using the Planets NM skin in standard, and the kitafuma 1.5 skin in taiko.

### Version

2024.519.1-lazer

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/15370667/compressed-logs.zip)

**Comment by bdach** [comment]
please attach video evidence of this & reproduction steps if possible

**Reply by EthanCannoy** [reply]
Added some repro information. will attach video evidence when I play next - assuming the issue comes through on video.
have validated it to be unlikely to be a hardware issue, as other audio playing on my system when the issue occured didn't increase in volume, and changing audio device did not fix the issue.
